lib/AST/Decl.cpp

```diff
@@ -5042,3 +5042,20 @@ getObjCNameForSwiftDecl(const ValueDecl *VD, DeclName PreferredName){
     return {Ctx.getIdentifier(getNameForObjC(VD)), ObjCSelector()};
   }
 }
+
+bool swift::objc_translation::
+isVisibleToObjC(const ValueDecl *VD, Accessibility minRequiredAccess,
```

Comment:
I would suggest either moving it to some other file, in lib/IDE/, or making it a method on ValueDecl instead of a top-level function. Just feels weird to see 'swift::objc_translation' in this file.